File: app/src/main/java/com/shmedo/mcloudapp/profile/callback/IOTCommandCallback.java
package com.shmedo.mcloudapp.profile.callback;

import android.bluetooth.BluetoothDevice;

import androidx.annotation.NonNull;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  12/7/20 <br/>
 * 描述：     TODO
 */
public interface IOTCommandCallback {

    void onResponseReceived(@NonNull final BluetoothDevice device, final String result);
}

File: lib_ble/src/main/java/com/shmedo/lib/ble/communicate/parser/IOTCommandCallback.kt
package com.shmedo.lib.ble.communicate.parser

import android.bluetooth.BluetoothDevice

/**
 * 创建者：gonghe
 *
 * 创建时间：2023/9/6
 *
 * 描述： TODO
 *
 *
 */
interface IOTCommandCallback {
    fun onResponseReceived(
        device: BluetoothDevice,
        cmdResult: String = "",
        cmdResultList: List<String> = listOf()
    )
}